client/src/modules/accounts/accounts.html

```diff
@@ -38,6 +38,20 @@
         </a>
       </div>
 
+      <div class="toolbar-item" ng-if="AccountsCtrl.showHiddenAccounts">
```

Comment:
Can we put this in the menu instead?  That way we don't run into issues on smaller screens:

![accountsweirdness](https://user-images.githubusercontent.com/896472/41820340-4b6aa7b4-77c8-11e8-91fe-6dac95f42f5f.PNG)
_Fig 1: Accounts Management on a small screen_

If you do leave it as a button, then put `class="hidden-xs"` on the `<span translate>` labels.